### ISSUE TITLE
(py4cl2:save-config) comes with errors

### ISSUE THREAD

**Issue by cl-03** [digikar99/py4cl2#6]
The device of pathname of QUICKLISP is "C",but the error signs "E":
typed at REPL:
`CL-USER> (py4cl2:save-config)`
 errors:
```
Error opening #P"e:/Users/chuangxiu.chen/quicklisp/local-projects/py4cl2-2.4.1/src/../.config"
   [Condition of type SB-INT:SIMPLE-FILE-ERROR]
```
the path of quicklisp is #P"c:/Users/chuangxiu.chen/quicklisp/local-projects/py4cl2-2.4.1/src/../.config" in fact.

**Comment by digikar99** [comment]
I'll look into this around the weekend; for the time being, does manually editing the config file at `c:/Users/chuangxiu.chen/quicklisp/local-projects/py4cl2-2.4.1/.config` and restarting lisp work for you?

**Comment by cl-03** [comment]
sorry,It still couldn't work

**Comment by cl-03** [comment]
the package PY4CL2 has replaced PY4CL, isn't it？

**Comment by digikar99** [comment]
> The device of pathname of QUICKLISP is "C",but the error signs "E":

Nope, unable to reproduce; I'm using [portacle](https://portacle.github.io/) though. Does it yield the same error if you simply write to some file?

```lisp
(with-open-file (f "/path/to/file" :if-does-not-exists :create
                                   :if-exists :supersede
                                   :direction :output)
  (write-line "hello world" f))
```

Tests intended to pass do pass on v2.6.0, perhaps except the test `with-python-output-stress-test`.

***

> the package PY4CL2 has replaced PY4CL, isn't it？

Sort of - I had to rename it primarily because of the incompatibility induced by the function argument importing. 
This should cause enough breaking changes for users. 

But then I went ahead and introduced more breaking changes, some aesthetic, some semantic. So, the current plan seems to be to provide bug-fixes and possibly backward-compatible changes to py4cl, and any breaking changes would be done here. I'm betting on: the earlier things break, the better it'd be for the longer run. And versioning exists as well, in case too many things break.
